.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "resources/public/contracts/src/ens-repo"]
+	path = resources/public/contracts/src/ens-repo
+	url = https://github.com/ensdomains/ens.git
```

Comment:
I take it it's not possible to have a specific commit in this file?

Reply:
In this file no, this is just an information about which repo is the submodule. However, the submodule acts as a normal git repo inside the project - Meaning that it has notion of branches and commits internally and the branch of submodule is fixed in the project. If you bump the submodule commit, you WILL commit it as change (smthng like this https://github.com/Siegrift/react-diagrams-app/commit/124d411300f0157814765023f458fbecc36872fd#diff-e7e55e6a474cad651590585e3553473c930a9b73f8582b6ff99e95a0b698dd52)